### PR TITLE
Support for modifying linux groups

### PR DESCRIPTION
**Pull request by arichardet**
- Ability to modify a group name and/or gid

### CODE REVIEW

**Comment by arichardet** [comment]
#279

File: resource/group/preparer.go [hunk 1]
```diff
@@ -32,6 +32,10 @@ type Preparer struct {
 	// Name is the group name.
 	Name string `hcl:"name" required:"true"`
 
+	// NewName is used when modifying a group.
+	// The group Name will be changed to NewName.
+	NewName string `hcl:"new_name"`
```
Comment (ryane) [inline resource/group/preparer.go]: need to regenerate docs

Comment (arichardet) [inline resource/group/preparer.go]: done

**Comment by ryane** [comment]
lgtm

**Pull request merged**